models/games.models.js

```diff
@@ -1,9 +1,21 @@
 const db = require("../db/connection");
+const reviews = require("../db/data/test-data/reviews");
 
 const fetchCategories = () => {
   return db.query(`SELECT * FROM categories;`).then((categories) => {
     return categories.rows;
   });
 };
 
-module.exports = fetchCategories;
+const fetchReviewById = (id) => {
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id = $1;`, [id])
+    .then((review) => {
+      if (!review.rows.length) {
+        return Promise.reject({ msg: "not found" });
```

Comment:
suggest you include a `status: 404` in your promise reject object, so `handleOtherErrors` can be more dynamic